sdk/include/reactos/shellutils.h

```diff
@@ -94,6 +94,35 @@ inline BOOL _ROS_FAILED_HELPER(HRESULT hr, const char* expr, const char* filenam
 } /* extern "C" */
 #endif /* defined(__cplusplus) */
 
+static inline UINT
+SHELL_ErrorBoxHelper(HWND hwndOwner, UINT Error)
```

Comment:
```suggestion
SHELL_ErrorBoxHelper(HWND hwndOwner, UINT n StringID, UINT Error)
```

Reply:
That is just retarded. You are just moving the problem now.

Or you are suggesting that shell32, browseui, shlwapi and who knows what else, are all are going to need their own duplicate error string in their resources?

Reply:
Sorry, that was user32's one. Ignore it.